### PR TITLE
Modify model parameters in Spark LLMs and zhipuai LLMs

### LINKED ISSUE #8053: Please change the Spark series model to the official name
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

The Spark series models in dify do not use the official naming scheme, and no pro-128k models are added. I hope to synchronize with the official.

### 2. Additional context or comments

_No response_

### 3. Can you help us with this feature?

- [x] I am interested in contributing to this feature.

### PR DESCRIPTION
**Pull request by AAEE86**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [ ] Please open an issue before creating a PR or link to an existing issue
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes https://github.com/langgenius/dify/issues/8053

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by AAEE86** [comment]
link #8053

File: api/core/model_runtime/model_providers/spark/llm/Spark-4.0Ultra.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+model: Spark-4.0Ultra
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark-4.0Ultra.yaml]:
```suggestion
model: spark-4.0-ultra
```

File: api/core/model_runtime/model_providers/spark/llm/Spark-4.0Ultra.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+label:
+  en_US: Spark 4.0Ultra
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark-4.0Ultra.yaml]:
```suggestion
  en_US: Spark 4.0 Ultra
```

File: api/core/model_runtime/model_providers/spark/llm/Spark-Lite.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+model: Spark-Lite
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark-Lite.yaml]:
```suggestion
model: spark-lite
```

File: api/core/model_runtime/model_providers/spark/llm/Spark-Max.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+model: Spark-Max
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark-Max.yaml]:
```suggestion
model: spark-max
```

File: api/core/model_runtime/model_providers/spark/llm/Spark-Pro-128K.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+model: Spark-Pro-128K
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark-Pro-128K.yaml]:
```suggestion
model: spark-pro-128k
```

File: api/core/model_runtime/model_providers/spark/llm/Spark-Pro-128K.yaml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+label:
+  en_US: Spark Pro-128K
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark-Pro-128K.yaml]:
```suggestion
  en_US: Spark Pro 128K
```

File: api/core/model_runtime/model_providers/spark/llm/Spark-Pro.yaml [hunk 7]
```diff
@@ -0,0 +1,33 @@
+model: Spark-Pro
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark-Pro.yaml]:
```suggestion
model: spark-pro
```

**Review by crazywoola — CHANGES_REQUESTED** [review]
Please change the filename to lowercase as well :)

**Review by laipz8200 — APPROVED** [review]
LGTM

**Review by crazywoola — APPROVED** [review]

**Pull request merged**